packages/frontend-core/src/utils/jsonFormatter.ts

```diff
@@ -0,0 +1,71 @@
+import { JSONValue } from "@budibase/types"
+
+export type ColorsOptions = {
+  keyColor?: string
+  numberColor?: string
+  stringColor?: string
+  trueColor?: string
+  falseColor?: string
+  nullColor?: string
+}
+
+const defaultColors: ColorsOptions = {
+  keyColor: "dimgray",
+  numberColor: "lightskyblue",
+  stringColor: "lightcoral",
+  trueColor: "lightseagreen",
+  falseColor: "#f66578",
+  nullColor: "cornflowerblue",
+}
+
+const entityMap: Record<string, string> = {
```

Comment:
It might be better not having the `Record<string, string>`, so the type gets properly typed automatically